integration_tests/cosmoscli.py

```diff
@@ -30,14 +30,19 @@ def __init__(
     ):
         self.data_dir = data_dir
         genesis_path = self.data_dir / "config" / "genesis.json"
+        self.raw = ChainCommand(cmd)
         if genesis_path.exists():
             self._genesis = json.loads(genesis_path.read_text())
             self.chain_id = self._genesis["chain_id"]
         else:
             self._genesis = {}
             self.chain_id = chain_id
+            # avoid client.yml overwrite flag in textual mode
```

Comment:
node url from flag get overwritten by client.toml in textual that need [query](https://github.com/cosmos/cosmos-sdk/blob/v0.53.4/x/auth/tx/config/config.go#L213) denom, I need swap [ReadFromClientConfig](https://github.com/MANTRA-Chain/mantrachain/blob/release/v5.0.0-rcx/cmd/mantrachaind/cmd/root.go#L78) and [ReadPersistentCommandFlags](https://github.com/MANTRA-Chain/mantrachain/blob/release/v5.0.0-rcx/cmd/mantrachaind/cmd/root.go#L73) or update default `127.0.0.1:26657` to livenet url in client.yml, and`viper.BindPFlags` seems correct though.